addons/project/controllers/portal.py

```diff
@@ -353,7 +353,7 @@ def portal_my_tasks(self, page=1, date_begin=None, date_end=None, sortby=None, f
                                          key=lambda item: item[1]["sequence"]))
 
         searchbar_filters = {
-            'all': {'label': _('All'), 'domain': []},
+            'all': {'label': _('All'), 'domain': [('project_id', '=', False)]},
```

Comment:
```suggestion
            'all': {'label': _('All'), 'domain': [('project_id', '!=', False)]},
```
The domain should also be updated in `_prepare_home_portal_values` in `odoo/addons/project/controllers/portal.py`, else the displayed values will be incorrect